pythonforandroid/bootstraps/common/build/templates/build.tmpl.gradle

```diff
@@ -96,6 +96,13 @@ android {
         }
     }
 
+    {% if args.enable_androidx %}
+    ext {
```

Comment:
Would it be preferable to put the `if`/`endif` *inside* the `ext` block (which would exist, but empty by default, and this way, more options could be added later in that block if needed?

Just a thought, it's certainly not a problem and it can be changed later when we actually have more options to put in there.